hazelcast-vnet/README.md

```diff
@@ -0,0 +1,14 @@
+# Deploy a Hazelcast Cluster
```

Comment:
A cluster doesn't necessarily need to be restricted to a vnet. For example 2 vnets and a gateway between the two is also valid.

There's also talk of another hazelcast template with VM scalesets in a couple months after the javasdk and plugin supports it.

So `vnet` indicates these are VMs within the same vnet forming a hazelcast cluster rather than a scaleset.